vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -92,6 +93,9 @@ const (
 	DockerUSocket           = "unix:///var/run/docker.sock"
 	defaultSleepIntervalSec = 1
 	dockerConnTimeoutSec    = 2
+	refCountDelayMin        = 2
```

Comment:
Can you add "Sec" postfix like other variables for e.g. refCountDelayMinSec & refCountDelayMaxSec